### PR TITLE
Use consistent API and order for title and sides

### PR DESCRIPTION
**Pull request by deathbeam**
Use same API for ordering title and sidebar and order them in the swing
components and not in events as that is unreliable.

Closes #4476

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

**Comment by Adam-** [comment]
Does this change the titlebar button order back to pre-1.4.9 or just make it deterministic?

**Comment by Abextm** [comment]
Is there a reason that the title toolbar buttons are no longer separated from the plugin toolbar buttons?

**Comment by deathbeam** [comment]
In sidebar with custom window chrome disabled? Hmm.., I can fix that. Should be same as pre-1.4.9 @Adam-

**Comment by deathbeam** [comment]
Updated

File: runelite-client/src/main/java/net/runelite/client/ui/ClientPluginToolbar.java [hunk 1]
```diff
@@ -53,33 +61,38 @@
...
 	}
 
-	public void removeComponent(final NavigationButton button)
+	private void update()
 	{
-		final Component component = componentMap.remove(button);
+		removeAll();
 
-		if (component != null)
+		final AtomicBoolean isDelimited = new AtomicBoolean(false);
```
Comment (Abextm) [inline runelite-client/src/main/java/net/runelite/client/ui/ClientPluginToolbar.java]: Using AtomicX it typically a good hint that you shouldn't use a lambda

**Comment by deathbeam** [comment]
Updated

**Pull request merged**